about/index.html

```diff
@@ -31,42 +31,37 @@
     </div>
 </section>
 
-<section class="features-section">
+<section class="principles-section">
     <div class="container-fluid content-holder">
-        <h1 class="light section-title">Features</h1>
-        {% include feature-section.html %}
-    </div>
-</section>
-
-<section class="story-section">
-    <div class="container-fluid content-holder">
-        <h1 class="bold section-title">The story behind Spine</h1>
+        <h2 class="bold section-title">Principles</h2>
         <div class="row">
             <div class="col-sm-7">
```

Comment:
To make the same Principles text wrap as the intro section does, we need to change the class `col-sm-7` to `col-lg-7`.